AWS/parse_lambda/bobae_parse.py

```
@@ -0,0 +1,184 @@
+import json
+import boto3
+import pandas as pd
+import io
+from bs4 import BeautifulSoup
+import re
+from datetime import datetime, timedelta
```

Ruff (0.8.2)

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)